backend/backend/handlers/metadata/__init__.py

```diff
@@ -0,0 +1,115 @@
+
+import boto3
+import json
+import logging
+import os
+import traceback
+
+# region Logging
+
+LOG_LEVEL = os.getenv("LOG_LEVEL", "INFO")
+logger = logging.getLogger()
+
+if logger.hasHandlers():
+    # The Lambda environment pre-configures a handler logging to stderr. If a handler is already configured,
+    # `.basicConfig` does not execute. Thus we set the level directly.
+    logger.setLevel(LOG_LEVEL)
+else:
+    logging.basicConfig(level=LOG_LEVEL)
+
+# endregion
+
+def mask_sensitive_data(event):
+    # remove sensitive data from request object before logging
+    keys_to_redact = ["authorization"]
+    result = {}
+    for k, v in event.items():
+        if isinstance(v, dict):
+            result[k] = mask_sensitive_data(v)
+        elif k in keys_to_redact:
+            result[k] = "<redacted>"
+        else:
+            result[k] = v
+    return result;
+
+def build_response(http_code, body):
+    return {
+        "headers": {
+            "Cache-Control": "no-cache, no-store", # tell cloudfront and api gateway not to cache the response
+            "Content-Type": "application/json",
+        },
+        "statusCode": http_code,
+        "body": body,
+    }
+
+
+region = os.environ['AWS_REGION']
```

Comment:
I think we may not need this, in lambda boto3 automatically picks up the right region